monkey/monkey_island/cc/next_ui/src/_components/alerts/ErrorAlerts.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Alert from '@/_components/alerts/Alert';
+import { Severity } from '@/_components/lib/severity';
+import classes from './errorAlerts.module.scss';
+
+export interface ErrorAlertsProps {
+    errors: string[];
+}
+
+const ErrorAlerts = (props: ErrorAlertsProps) => {
```

Comment:
We could also make this a heterogeneous AlertsList component and provide the Severity as a property

Reply:
Good idea on paper, but I think in practice we'll either have an error list or a mixed alert list (success/info/warning). Until we need a mixed alert list, I think it's best to keep this simple interface. We could rename to `ErrorList` maybe?